### PR TITLE
DOCSP-28808: conversion operations

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-28808
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-28808-conversion-operations/fundamentals/aggregation/aggregation-expression-operations/#conversion-operations)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/fundamentals/aggregation/aggregation-expression-operations.txt [hunk 1]
```diff
@@ -192,6 +192,77 @@ the Query API:
...
+     - :manual:`$dateToString </reference/operator/aggregation/dateToString/>`
+
+   * - `parseDate() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#parseDate()>`__
+     - :manual:`$dateFromString </reference/operator/aggregation/dateFromString/>`
+
+   * - `parseInteger() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#parseInteger()>`__
+     - :manual:`$toInt </reference/operator/aggregation/toInt/>`
+
+   * - `millisecondsAsDate() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlInteger.html#millisecondsAsDate()>`__
+     - :manual:`$toDate </reference/operator/aggregation/toDate/>`
```
Comment (DBirtolo-mdb) [inline source/fundamentals/aggregation/aggregation-expression-operations.txt:227]: I'd recommend putting these in alphabetical order so it is easier for readers to follow.

```suggestion
   * - `millisecondsAsDate() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlInteger.html#millisecondsAsDate()>`__
     - :manual:`$toDate </reference/operator/aggregation/toDate/>`

   * - `parseDate() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#parseDate()>`__
     - :manual:`$dateFromString </reference/operator/aggregation/dateFromString/>`

   * - `parseInteger() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#parseInteger()>`__
     - :manual:`$toInt </reference/operator/aggregation/toInt/>`
```

File: source/fundamentals/aggregation/aggregation-expression-operations.txt [hunk 2]
```diff
@@ -192,6 +192,77 @@ the Query API:
...
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+
+Suppose you want to calculate students' fifth reunion years based on
+their graduation years, which are stored as strings. You want to add a
```
Comment (DBirtolo-mdb) [inline source/fundamentals/aggregation/aggregation-expression-operations.txt:232]: [suggestion]

For readability, I'd recommend a slight rewording. You could also break this into two sentences, but that's totally up to you.

```suggestion
Suppose you have a collection of students with their graduation years
stored as strings and want to calculate the years for their fifth reunions.
You want to add a
```

**Review by DBirtolo-mdb — CHANGES_REQUESTED** [review]
Nice start. Just a couple of minor tweaks recommended.

**Review by katcharov — CHANGES_REQUESTED** [review]

**Review by katcharov — APPROVED** [review]
LGTM

**Pull request merged**